Libraries/Types/CoreEventTypes.js

```diff
@@ -26,6 +26,12 @@ export type SyntheticEvent<T> = $ReadOnly<{|
   persist: () => void,
   target: ?number,
   timeStamp: number,
+  touchHistory: $ReadOnly<{|
```

Comment:
I am not sure - I tried a few events out, and they all seemed to have it - but those were all touch-related, so that it probably why. I will make it nullable, but will require some more modifications to panresponder

Reply:
Looks like this is where it is set: https://github.com/facebook/react-native/blob/5491c3f942430982ce9cb6140ed1733879ed3d1d/Libraries/Renderer/oss/ReactNativeRenderer-dev.js#L1629-L1638

Reply:
Want to add a matching ResponderSyntheticEvent to CoreEventTypes?

```
export type ResponderSyntheticEvent<T> = $ReadOnly<{|
  ...SyntheticEvent<T>
  // touchHistory...
|}>

Reply:
Tried that, and it works great! 👍